{{cookiecutter.project_name}}/.github/workflows/ci-pybind11.yml

```diff
@@ -59,4 +59,7 @@ jobs:
         run: python -m pip install -e .[test]
 
       - name: Test package
-        run: python -m pytest -ra
+        run: python -m pytest -ra --cov={{ cookiecutter.project_name }}
```

Comment:
`{{ cookiecutter.project_name }}` does not work in `yaml` files

Reply:
Are there any downsides? Otherwise, that would likely work well. We disable `{{}}` subs since it interferes with `${{}}` subs from Actions.

Reply:
We could also add it to nox and remove it from here.

Reply:
> Are there any downsides?

`pytest-cov` generates a coverage report encapsulating all the sub-packages present in the current directory (for instance, the `tests` folder), so not ideal.

I think adding it `nox` would be the best workaround here. But then, should these workflow files use `nox` to run the tests?